samples/Beyer2024/create_paper_data.py

```diff
@@ -40,7 +41,7 @@
 labels_fig7=["7a", "7b", "7c"]
 
 if fig_label in labels_fig7 and not plot:
-    script_path=pmb.get_resource(f"samples/Beyer2024/peptide.py")
+    script_path=pmb.get_resource("samples/Beyer2024/peptide.py")
```

Comment:
@jngrad @pm-blanco  It's not related to the changes on the PR but we need to fix this. In line 43, Is there a reason for `and not plot`? Because if we run the script with `--plot` the `sequence` of the peptides is not defined on that section (line 183) and the script crashes.

Reply:
I am not too familiar with this script, so I would trust @pm-blanco's opinion here.

Looking at the PR diff again, I also just stumbled upon the following oddity:
https://github.com/pyMBE-dev/pyMBE/blob/d10d17236217f71c2c13d8c7816e065c7d9e21e2/pyMBE.py#L1859-L1860
I'm pretty confident `sys.exit(1)` would be more appropriate than raising an exception. This would also be more consistent with other instances of `if hard_check:` in the code. Right now, it's unclear to me how a pyMBE simulation is supposed to react when that exception is raised, because it won't be caught by `except Exception` (since it derives from `BaseException`), and it sends a signal as a side-effect, which propagates through subprocess calls in non-trivial ways. For example, subprocess can convert it from a signal to an exception! Consider the following example:
```python

>>> import subprocess
>>> subprocess.check_output(["python3", "-c", "raise ValueError()"])
Traceback (most recent call last):
subprocess.CalledProcessError: Command '['python3', '-c', 'raise ValueError()']' returned non-zero exit status 1.
>>> subprocess.check_output(["python3", "-c", "raise KeyboardInterrupt()"]) # signal becomes and exception!
Traceback (most recent call last):
subprocess.CalledProcessError: Command '['python3', '-c', 'raise KeyboardInterrupt()']' died with <Signals.SIGINT: 2>.
```

The [`KeyboardInterrupt`](https://docs.python.org/3/library/exceptions.html#KeyboardInterrupt) documentation has a disclaimer to deter people from using it in the way we do.